build-files/krona/README.md

```diff
@@ -0,0 +1,35 @@
+
+# Krona container
+
+Main tool: [Krona](https://github.com/marbl/Krona)
+  
+Code repository: <https://github.com/marbl/Krona/tree/master/KronaTools>
+
+Additional tools:
+
+- krona 2.8.1
+- krakentools d4a2fbe
+- pandas 2.2.3
+- bioptyhon 1.84
+- miniconda (latest)
+
+Basic information on how to use this tool:
+- executable: various perl scripts
```

Comment:
Is it possible to list them? table2asn and a few other containers have a lot of executables, and it's often helpful to list which ones are included.

Reply:
Listed!